components/eamxx/src/control/atmosphere_driver.cpp

```diff
@@ -924,17 +931,18 @@ void AtmosphereDriver::set_initial_conditions ()
   // If a filename is specified, use it to load inputs on all grids
   if (ic_pl.isParameter("Filename")) {
     // Now loop over all grids, and load from file the needed fields on each grid (if any).
-    m_atm_logger->debug("    [EAMxx] Reading fields from file ...");
+    m_atm_logger->info("    [EAMxx] Reading fields from file ...");
     const auto& file_name = ic_pl.get<std::string>("Filename");
     for (const auto& it : m_field_mgrs) {
       const auto& grid_name = it.first;
+      //m_atm_logger->info("grd name for FM is "+grid_name);
       read_fields_from_file (ic_fields_names[grid_name],it.first,file_name,m_current_ts);
     }
-    m_atm_logger->debug("    [EAMxx] Reading fields from file ... done!");
+    m_atm_logger->info("    [EAMxx] Reading fields from file ... done!");
```

Comment:
You can have all "debug" messages print by changing the atm_log_level to "debug"
```
./atmchange atm_log_level=debug
```